src/trufflecon2020.html

```diff
@@ -71,18 +71,30 @@ <h2>Grow with the Blockchain Community</h2>
       </div>
     </div>
     <div class="row pt-4 text-center">
-      <div class="col-md-6 order-2 order-md-1 pt-3">
+      <div class="col-md-6 order-2 order-md-1 pt-3 pb-3">
         <p><h6>(sales end October 1st)</h6>
           Enjoy a tasting flight of hand-crafted bean-to-bar chocolates that include delicious milk, white, and dark varieties for every kind of chocolate lover! Janet and Joe, owners of J&J Chocolates, will walk us through tasting their chocolates while sharing how they make bean-to-bar chocolates and truffles.
         </p>
         <div>
-          <a class="btn btn-lg btn-block btn-truffle" href="#signup">I love me some chocolate</a>
+          <a class="btn btn-lg btn-block btn-truffle pb-2" href="#signup">I love me some chocolate</a>
         </div> 
       </div>
-      <div class="col-md-6 order-1 order-md-2 pt-3">
+      <div class="col-md-6 order-1 order-md-2 pt-3 pb-3">
         <img src="/img/trufflecon/swag-choc-display-6.png" class="img-fluid"> 
       </div>
-    </div>
+    </div><!--
+    <div class="row pt-4 text-center">
+      <div class="col-md-6 pt-3 pb-3">
+        <img src="/img/trufflecon/poker-asset2.svg" class="img-fluid" width="80%" alt="Charity Poker" />
+      </div>
+      <div class="col-md-6 pt-3 pb-3">
+        <p>
+            The blockchain space includes a multitude of organizations leveraging innovative technology to tackle some of the world’s biggest challenges. To highlight these efforts and encourage more people to wield their tech stacks for charitable causes, we’ve included a Blockchain for Social Good speaking track as part of this year’s TruffleCon. In addition, we’ll be hosting a charity poker event chock-full of well-known blockchain superstars to raise funds for non-profit organizations.        </p>
+        <div>
+          <a class="btn btn-lg btn-block btn-truffle pt-2 pb-2" href="#signup">I'M IN</a>
+        </div> 
+      </div>
+    </div>-->
```

Comment:
I'm personally not a fan of these commented out blocks. I'd personally rather they just be implemented in other branches waiting to be merged in. However, I don't have a stake in the game here, so 🤷 yall do what you want

Reply:
Yeah, I'd prefer to comment them out for now since they'll be added shortly. I have the worst luck when I stash files with github.